db/migrate/20150727085217_set_conditional_type_nullity.rb

```diff
@@ -0,0 +1,5 @@
+class SetConditionalTypeNullity < ActiveRecord::Migration
+  def change
+    change_column_null :course_conditions, :conditional_type, false
```

Comment:
This and the id column can be squashed manually into one file, schema.rb doesn't change.